data/dev-env/balancer.docker.toml

```diff
@@ -0,0 +1,6 @@
+api_key = "<randomly generate this>"
```

Comment:
Probably specify that this should be alphanumerical only so Chris/Mike don't run into the same problem I did.